src/pytomlpp/_io.py

```diff
@@ -0,0 +1,27 @@
+"""Python wrapper for Toml++ IO methods."""
+
+from . import _impl
+
+
+def dump(data, fl):
```

Comment:
Maybe later we can add type annotations to these python functions but for now, it is okay.

Reply:
Type annotations require Python 3.5, up to you